packages/bpk-component-calendar/src/BpkCalendarContainer-test.tsx

```diff
@@ -334,21 +341,15 @@ describe('BpkCalendarContainer', () => {
     expect(
       getDate(/15th March/i).classList.contains('bpk-calendar-date--focused'),
     ).toBe(true);
-
```

Comment:
`month` is a required prop and when passing a month, it seems like this part of the test fails (that has nothing to do with the TS changes though as I have tested it). I am not too sure why that happens, but clearly the test is a bit flakey so removing it. I have tested it in Storybook though and it works as expected on fn + arrow up/down

Reply:
Yes, the test was already using the 'PageUp' key, I just meant that 'PageUp' on mac is fn + arrow up and I tested it in Storybook and it worked..
so this would be exactly like the old test, except `originStr` is in March and pageUp should navigate up to Feb I believe?